contracts/src/protocol/OmniPortalConstants.sol

```diff
@@ -15,12 +21,18 @@ contract OmniPortalConstants {
     /// @notice Minimum allowed xmsg gas limit
     uint64 public constant XMSG_MIN_GAS_LIMIT = 21_000;
 
-    /// @notice Numerator of the fraction of total validator power required to accept an XSubmission. Ex 2/3 -> 2
-    uint8 public constant XSUB_QUORUM_NUMERATOR = 2;
-
-    /// @notice Denominator of the fraction of total validator power required to accept an XSubmission. Ex 2/3 -> 3
-    uint8 public constant XSUB_QUORUM_DENOMINATOR = 3;
-
     // TODO: make gas limits admin-configurable
     // TODO: make quorum admin-configurable
+
+    // @dev xmsg.destChainId for "broadcast" xcalls, intended for all portals
+    uint64 internal constant _BROADCAST_CHAIN_ID = 0;
+
+    // @dev xmsg.sourceChainId of xmsgs from Omni's consensus chain
+    uint64 internal constant _CCHAIN_ID = 0;
+
+    // @dev xmg.sender for xmsgs from Omni's consensus chain
+    address internal constant _CCHAIN_SENDER = address(0);
+
+    // @dev xmsg.to for xcalls to be executed on the portal itselfj
+    address internal constant _VIRUTAL_PORTAL_ADDRESS = address(0);
```

Comment:
We might want an address other than 0 here. Could end up being used for something else. For example, depending on how our "cross chain deployment factory" ends up working, we may want to allow CREATE txs to the 0 address. (tho I'd imagine that will probably be through some factory contract)